public_html/wp-content/mu-plugins/blocks/source/block/organizers/inspector-controls.js

```diff
@@ -2,7 +2,7 @@
  * WordPress dependencies
  */
 const { PanelBody, PanelRow, SelectControl, ToggleControl } = wp.components;
-const { InspectorControls }                                 = wp.editor;
+const { InspectorControls: Inspector }                      = wp.editor;
```

Comment:
I thought about this too. I could flip it so ours is the one named just `Inspector`, but then I'd probably want to rename the file to just `inspector.js` as well.

On the other hand, the Gutenberg element really is the whole Inspector Sidebar, while ours is just a component that adds some additional inspector controls, so semantically this makes more sense to me, even if we're renaming the Core comonent (which, yes, is due to a naming conflict)

Reply:
🤔 

Well, to further complicate things, to me `Inspector` makes it sound like it's the entire `BlockInspector`, rather than just the controls for it, so I'm not sure that that name works well for either `wp.editor.InspectorControls` or our `OrganizerInspectorControls`.

Reply:
If it'd be more practical/semanic to rename `wp.editor.InspectorControls` rather than our custom components, then maybe it should be named `BlockInspectorControls` ?

That still feels odd to me, though, because I perceive the platform's components as (conceptually) immutable, whereas the custom stuff we build on top of that is malleable, and created in light of that platform's prior existence. 

To draw an analogy to PHP, I don't think we'd ever to this:

```php
<?php

namespace WordCamp\Foo;
use function wp_insert_post as insert_post;

function wp_insert_post( $arr ) {
    // do some custom wrapper processing on $arr
    insert_post( $arr );
}
```

...instead, we'd just

```php
<?php

namespace WordCamp\Foo;

function insert_post( $arr ) {
    // do some custom wrapper processing on $arr
    wp_insert_post( $arr );
}
```

Reply:
<a href="https://www.commitstrip.com/en/2015/10/27/one-of-the-coders-hardest-problems/"><img src="https://www.commitstrip.com/wp-content/uploads/2015/10/Strip-Trouver-le-nom-de-variable-english650-final.jpg" alt="There are only 2 hard things in computer science..." /></a>

😉

Reply:
💯

Reply:
Good point that platform component names should probably be immutable. What about eschewing "Inspector" altogether and going with something like `BlockSidebar`? Same idea, and it still follows the convention we're using with `BlockControls` and `BlockContent`...

Reply:
🤔 I don't love it, since "inspector" feels more specific and meaningful.

🤔, maybe `InspectorControlContents` ? It's a bit of a mouthful, but kinda fits. It's not specific to Organizers, but I guess the intent is to make it more generic inside the component anyway. We could even shorter it to `ControlContents` since the "inspector" part is mostly obvious from it being wrapped in an `<InspectorControl>`. 

🤔 I don't love that either, and it could also conflict if we have other types of controls inside the parent component.

🤷‍♂Oh well, it's a crapshoot, pick the one you like best and :shipit:  😄

Reply:
Maybe not a popular decision, but a decision nonetheless: 8ec11263bd5c149d52dc241d398cc6cecde5e2c1